source/fundamentals/enterprise-authentication.txt

```diff
@@ -219,8 +219,8 @@ using your directory-server username and password.
 The following examples specify the authentication mechanism using the following
 placeholders:
 
-- ``<username>``: Your LDAP username
-- ``<password>``: Your LDAP password
+- ``<db_username>``: Your LDAP username
+- ``<db_password>``: Your LDAP password
```

Comment:
Q: Similar question as above: are these different from the database username and password? If so, I'd remove the <db_> part of the placeholder and just keep as is. If not, I'd change to:
```suggestion
- ``<db_username>``: Your database username
- ``<db_password>``: Your database password
```

Reply:
Can revert both this and the other commit since it sounds like it's separate.